roles/os_firewall/tasks/firewall/iptables.yml

```diff
@@ -21,7 +21,7 @@
   ignore_errors: yes
 
 - name: Install iptables packages
-  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
```

Comment:
Good catch. Too bad maintainer closed original bug instead of routing it properly.